src/main/webapp/app/modules/canvas/visualization/util/d3-utils.ts

```diff
@@ -0,0 +1,19 @@
+export const getDefaultColorset = () => {
+  return []; //colorset;
+};
+
+export const getNames = arr => {
+  arr = arr.sort(function (a, b) {
```

Comment:
create a reusable function for the below code.

arr = arr.sort(function (a, b) {
    return a.order - b.order;
});

Reply:
this file is not in use now I have remove this file so ignore this file